<launch>

<node name="gimbal_driver" pkg="dynamixel_controllers" type="run_gimbal.py" output="screen">
  <param name="max_angle" type="double" value="30.0"/>
  <param name="min_angle" type="double" value="-15.0"/>
</node>

<include file="$(find dynamixel_controllers)/launch/dynamixel_manager.launch"/>
<include file="$(find dynamixel_controllers)/launch/position.launch"/>

</launch>
